### PR TITLE
Enable 'make test'

### PR DESCRIPTION
**Pull request by Billy99**
While working on PR #20, I was unable to run 'make test' in my local sandbox. The following set of commits fixes the install-deps, gofmt and golint issues I encountered.

### CODE REVIEW

File: client.go [hunk 1]
```diff
@@ -49,9 +49,9 @@ var (
 const (
 	defaultTCPAddress  = "127.0.0.1:6640"
 	defaultUnixAddress = "/var/run/openvswitch/ovnnb_db.sock"
-	SSL             = "ssl"
```
Comment (vtolstov) [inline client.go:52]: why you unexport this const ?

File: config.go [hunk 2]
```diff
@@ -20,8 +20,7 @@ import (
 	"crypto/tls"
 )
 
-type Config struct {
-	Addr         string
-	TLSConfig    *tls.Config
+type config struct {
```
Comment (vtolstov) [inline config.go]: why unexported?

**Review by vtolstov — CHANGES_REQUESTED** [review]

File: client.go (hunk 1 above)
Comment (Billy99) [reply client.go:52]: gofmt wasn't happy with it all CAPS. I couldn't find it used outside of project so made local. I can revert and add comments for extern variables/const if intended to be externed.

File: config.go (hunk 2 above)
Comment (Billy99) [reply config.go]: Same as previous.

Comment (vtolstov) [reply config.go]: it used in go-ovn repo

Comment (Billy99) [reply config.go]: I could not find `libovsdb.Config` used anywhere in go-ovn. go-ovn has it's own version of `Config` defined in `github/eBay/go-ovn/config.go`.
However, I really don't care. `golint` just wants a comment with each exported type, so I will revert my change and add a comment.

File: client.go (hunk 1 above)
Comment (Billy99) [reply client.go:52]: s/gofmt/golint/

golint just wants a comment with each exported type, so I will revert my change and add a comment.

**Comment by hzhou8** [comment]
LGTM

**Review by vtolstov — APPROVED** [review]

**Pull request merged**